website/client-old/js/services/questServices.js

```diff
@@ -104,6 +104,9 @@ angular.module('habitrpg')
             Groups.data.party = party;
             $state.go('options.social.party');
             resolve();
+            if ($state.current.name === "options.social.party") {
+              $state.reload();
```

Comment:
While poking around for doing something like this I figured out that the $scope.go line can be changed to
`$state.go('options.social.party', {}, {reload: true});`

and then you don't need the if statement at all, as it "forces" a reload. Is this preferred or should I keep the if statement?